### PR TITLE
Added a slider to the Prune config to specify thedays to prune, default to 7

### PR DESCRIPTION
**Pull request by jadijadi**
Also added this new setting (prune_days) to the save and blocking_load functions and update hover texts accordingly.

### CODE REVIEW

**Comment by mikedilger** [comment]
Thanks for the submission.  Unfortunately this is a bad time to merge PRs that change structures as saved.  I have a very extensive conversion to LMDB in process.  I will come back to this and make a migration to modify the Settings struct after that work is complete.

File: src/db/mod.rs [hunk 1]
```diff
@@ -148,18 +148,20 @@ fn upgrade(db: &Connection, mut version: usize) -> Result<(), Error> {
...
+        db.execute_batch(&prune_sql_command)?;
         Ok::<(), Error>(())
     })
     .await??;
 
     GLOBALS
         .status_queue
         .write()
-        .write("Database prune has completed.".to_owned());
+        .write(format!("Database prune for {prune_days} has completed.").to_owned());
```
Comment (ArshiAAkhavan) [inline src/db/mod.rs:164]: `format` macro returns `String`. no need to do `.to_owned()` afterwards

Comment (Amirhossein2000) [inline src/db/mod.rs:164]:
```suggestion
        .write(format!("Database prune for {prune_days} has completed."));
```

**Comment by Amirhossein2000** [comment]
This PR is blocked by #431

**Review by onionj — APPROVED** [review]

**Comment by mikedilger** [comment]
This has been superceded by the settings branch which has merged into unstable.  I set the prune period slider to go from 7 days up to 360 days.

**Pull request closed without merging**